package.json

```diff
@@ -24,7 +24,7 @@
     "watch": "run-p watch:types watch:esbuild",
     "watch:types": "yarn build:types -w",
     "watch:esbuild": "node ./scripts/esbuild.js",
-    "dev": "yarn watch",
+    "dev": "yarn build:esm && yarn build:types && yarn watch",
```

Comment:
TODO: I need to revert this commit but was noticing some issues with dev watch behavior. Can circle back in one of our syncs.